jsonparse.ps1

```diff
@@ -0,0 +1,83 @@
+$location=$args[0]
+$repo=$args[1]
+$name=$args[2]
+
+$location
+$repo
+$name
+$repo.Title
+$JSON = (Get-Content $location) -join "`n" | ConvertFrom-Json
+
+$base = "https://github.com/"
```

Comment:
I think our replacement uri is being formed incorrectly.

I believe the current script will create a replacement uri similar to this: [https://github.com/sarif-sdk-8e0d2/src/Sarif/Extensions.cs](https://github.com/sarif-sdk-8e0d2/src/Sarif/Extensions.cs)
I don't think this url will resolve on github. 

I suspect this is the url you intended. It's the url to the commit version of the file on github.
[https://github.com/Microsoft/sarif-sdk/blob/8e0d243d1eb61cc726d638b909144a84576dc6a4/src/Sarif/Extensions.cs](https://github.com/Microsoft/sarif-sdk/blob/8e0d243d1eb61cc726d638b909144a84576dc6a4/src/Sarif/Extensions.cs)
I think this is the format using the appveyor environment variables.
[https://github.com/$env:APPVEYOR_REPO_NAME/blob/$env:APPVEYOR_REPO_COMMIT/src/Sarif/Extensions.cs](https://github.com/$env:APPVEYOR_REPO_NAME/blob/$env:APPVEYOR_REPO_COMMIT/src/Sarif/Extensions.cs)

That being said, we should consider using the  link to the raw file instead of the github url. I can see arguments for either, but this link will make it easier for consumers and, I think, more closely matches the intent of the link. 
[https://raw.githubusercontent.com/Microsoft/sarif-sdk/8e0d243d1eb61cc726d638b909144a84576dc6a4/src/Sarif/Extensions.cs](https://raw.githubusercontent.com/Microsoft/sarif-sdk/8e0d243d1eb61cc726d638b909144a84576dc6a4/src/Sarif/Extensions.cs)
This is the format using the appveyor environment variables.  
[https://raw.githubusercontent.com/$env:APPVEYOR_REPO_NAME/$env:APPVEYOR_REPO_COMMIT/src/Sarif/Extensions.cs](https://raw.githubusercontent.com/$env:APPVEYOR_REPO_NAME/$env:APPVEYOR_REPO_COMMIT/src/Sarif/Extensions.cs)


---
In reply to: [70169279](https://github.com/Microsoft/sarif-sdk/pull/420#discussion_r70169279) [](ancestors = 70169279)